base_layer/wallet_ffi/.venv/lib/python3.12/site-packages/_pytest/_py/path.py

```
@@ -0,0 +1,1475 @@
+        cannot be found, None is returned
+        Note: This is probably not working on plain win32 systems
+        but may work on cygwin.
+        """
+        if isabs(name):
+            p = local(name)
+            if p.check(file=1):
+                return p
+        else:
+            if paths is None:
+                if iswin32:
+                    paths = os.environ["Path"].split(";")
```

Ruff (0.11.9)

Use capitalized environment variable `PATH` instead of `Path`

Replace `Path` with `PATH`

(SIM112)